src/lib/Scenes/Sale/Components/RegisterToBidButton.tsx

```diff
@@ -107,10 +108,21 @@ export const RegisterToBidButtonContainer = createFragmentContainer(RegisterToBi
 })
 
 const tracks = {
-  auctionBidButtonTapped: ({ slug, status, contextType }: { slug: string; status: string; contextType: string }) => ({
-    action_name: "Tapped Register To Bid",
-    auction_slug: slug,
-    auction_state: status,
-    context_type: contextType,
-  }),
+  auctionBidButtonTapped: ({
+    contextScreenOwnerId,
+    contextScreenOwnerSlug,
+    contextType,
+  }: {
+    contextScreenOwnerId: string
+    contextScreenOwnerSlug: string
+    contextType: ScreenOwnerType
+  }) => {
+    const trackArgs: TappedRegisterToBidArgs = {
+      contextModule: contextType === "sale" ? ContextModule.auctionHome : ContextModule.auctionsInfo,
```

Comment:
[blocking] 2 suggestions in this line: (1) compare to the enum value instead of the string to avoid hardcoded strings, and (2) per @mikehrom we should use `ContextModule.aboutThisAuction` for the "about this auction" screen.

```suggestion
      contextModule: contextType === OwnerType.sale ? ContextModule.auctionHome : ContextModule.aboutThisAuction,
```

Reply:
[non-blocking] I also prefer to pass in an additional argument for the `contextModule` than to make a decision here based on `contextType`, but that's only a preference. If the caller is already making a decision about what to pass for `contextType`, it seems reasonable to me that it can also make a decision about what to pass for `contextModule`.

Reply:
you're absolutely true 🙏

Reply:
Submitting PRs after 8pm is never a good idea 🙈